src/components/DisplayComponents/Screen.js

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from "react";
+
+export default () => {
```

Comment:
The Display component was set up to do this, but good work. Also realizing you can export default directly on the function. I prefer to name it with a variable, just so I know what to call it, but it doesn't really make a difference.